share/admin.py

```diff
@@ -213,6 +214,19 @@ class HarvestLogAdmin(admin.ModelAdmin):
     list_filter = ('status', SourceConfigFilter, )
     list_select_related = ('source_config__source', )
     readonly_fields = ('harvest_log_actions',)
+    actions = ('restart_tasks', )
+
+    STATUS_COLORS = {
+        'created': 'blue',
+        'started': 'cyan',
+        'failed': 'red',
+        'succeeded': 'green',
+        'rescheduled': 'goldenrod',
+        'defunct': 'purple',
```

Comment:
I maybe should have thought about this before, but how does something become defunct?